setup.py

```diff
@@ -352,7 +361,7 @@ def load(filename):
         _cmake_build_dir, "deps", "magnum-bindings", "src", "python"
     )
 
-    if not has_magnum:
+    if not args.skip_reinstall_magnum or not has_magnum:
```

Comment:
```suggestion
    if not args.skip_install_magnum:
```

Maybe `--skip-install-magnum` instead and make it skip *always*, even if magnum doesn't seem to be installed? I would consider that a "pro" option where you want it to reliably skip installing magnum no matter what was the reason for `ImportError` being raised.

Reply:
Makes sense and simplifies logic :)

Reply:
I will also remove the yelling about pip.  The fact that things work with pip is kinda shadow supported and the yelling thing actually doesn't work as pip does some weird things with the environment so `has_magnum` is always false when installing via pip anyways.